app/src/test/java/com/losandroides/learn/domain/ItemMapperKtTest.kt

```diff
@@ -0,0 +1,42 @@
+package com.losandroides.learn.domain
+
+import com.losandroides.learn.data.network.item.model.ItemDTO
+import com.losandroides.learn.data.network.item.model.ItemsDTO
+import com.losandroides.learn.domain.model.Item
+import org.amshove.kluent.shouldBeEqualTo
+import org.junit.Test
+
+class ItemMapperKtTest {
+
+    @Test
+    fun `GIVEN empty DTO list WHEN ItemsDTO toDomain THEN return the expected domain list`() {
+        val itemsDTO = ItemsDTO(emptyList())
+        val items = emptyList<Item>()
+
+        val result = itemsDTO.toDomain()
+
+        result shouldBeEqualTo items
+    }
+
+    @Test
+    fun `GIVEN normal DTO list WHEN ItemsDTO toDomain THEN return the expected domain list`() {
+        val titles = listOf(
+            "1",
+            "2"
+        )
+        val itemsDTO = ItemsDTO(
+            listOf(
+                ItemDTO(titles[0]),
+                ItemDTO(titles[1])
+            )
+        )
+        val items = listOf(
+            Item(titles[0]),
+            Item(titles[1])
+        )
```

Comment:
I'll accept the PR, but this comment is only to do my bit on the knowledge of the community. 

I prefer to use the extensions of the collections, like `map` in this case. Well, we're only doing it for 2 items, but it could be useful for bigger lists.


```suggestion
        val itemsDTO = ItemsDTO(
            titles.map { title -> ItemDTO(title) }
        )
        val items = titles.map { title -> Item(title) }
```

Reply:
And a very good suggestion! Thank you very much, mate! 🚀